build.gradle

```diff
@@ -153,7 +153,7 @@ def versions = [
 ]
 
 dependencies {
-  implementation 'com.github.hmcts:ecm-common:0.1.132'
+  implementation 'com.github.hmcts:ecm-common:0.1.135'
```

Comment:
ethos service has version 0.1.136 of ECM-common. It should be same for both services.

Reply:
Updated now